src/Entities/Family.php

```diff
@@ -0,0 +1,154 @@
+<?php
+
+namespace Choredo\Entities;
+
+use Ramsey\Uuid\Uuid;
+
+/**
+ * Class Family
+ * @package Choredo\Entities
+ *
+ * @ORM\Entity
+ */
+class Family
+{
+    /**
+     * @var Uuid
+     *
+     * @ORM\Id
+     * @ORM\Column(type="uuid")
+     */
+    private $id;
```

Comment:
Would highly recommend making a behavior for this. You're going to get tired of typing it.